grpc.go

```diff
@@ -393,8 +393,10 @@ func (a *agentGRPC) StartContainer(ctx context.Context, req *pb.StartContainerRe
 		return nil, err
 	}
 
-	if status != libcontainer.Created {
-		return nil, fmt.Errorf("Container %s status %s, should be %s", req.ContainerId, status, libcontainer.Created)
+	if status != libcontainer.Created && status != libcontainer.Stopped {
```

Comment:
So we allow `Stopped` -> `Running` state transition for agent containers. I'm not sure if this can work because the libcontainer container process does not exist and thus libcontainer.Exec() does nothing.

Reply:
@sboeuf In which cases do we need to start a container that's in the `Stopped` state?

Reply:
My bad, with all those moving pieces, I have just realized that I don't need this anymore to make the code working.